.circleci/config.yml

```diff
@@ -21,4 +24,8 @@ workflows:
   version: 2
   build-and-deploy:
     jobs:
-      - build_and_test
+      - build_and_test:
+          filters:
```

Comment:
Is this necessary when using the ignore: above? Did you try without?

Reply:
k, ignoring the gh-pages branch does not work, as it would require a ci file in the branch (but handled by mkdocs automatically so no option to add that). Now just skipping for the docs ci by adding "[ci skip]" to the commit message (make gh-pages). Thanks for looking into this!